src/DataLayer/converters/MeasureConverter.ts

```diff
@@ -156,7 +157,7 @@ function convertArithmeticMeasureDefinition(definition: IArithmeticMeasureDefini
     const { arithmeticMeasure } = definition;
     return {
         arithmeticMeasure: {
-            measureIdentifiers: arithmeticMeasure.measureIdentifiers.splice(0),
+            measureIdentifiers: cloneDeep(arithmeticMeasure.measureIdentifiers),
```

Comment:
is this only one solution how to fix it? 

using cloneDeep looks to me like covering another issue... :/

Reply:
I need to create a new array with a copy of the elements in it. What other approaches would you suggest?